spacemake/cmdline.py

```diff
@@ -824,7 +842,6 @@ def spacemake_run(args):
         keepgoing=args["keep_going"],
         printshellcmds=args["printshellcmds"],
         config=config_variables,
-        rerun_triggers=["mtime"],
```

Comment:
why remove this? I added this line to restore the old snakemake behaviour. Current versions rerun even if you change source files or rules files, which is excessive.

Reply:
You're right, I removed this by accident when solving the merging problems.